### ISSUE TITLE
Donation

### ISSUE THREAD

**Issue by WhereJuly** [pawelgrzybek/siema#139]
Hi Pawel,

I sometimes use Siema in the commercial production. Do you have a donation link or the like? Paypal?

Thanks.

**Comment by pawelgrzybek** [comment]
Hi.

If you really want to support my hard work financially you can do it via https://www.paypal.me/pawelgrzybek.

I will be equally happy if you could share a Siema across your social media channels and recommend it to your front end friends. If you could share a URL to the project that you built with Siema it would be fantastic too! I am collecting a list of websites like this to promote it on a new section on a website (coming with next minor release).

Thanks a lot for your kind suggestion and constant support @WhereJuly

**Comment by WhereJuly** [comment]
Hi Pawel. Welcome :) I did send some humble amount. However many a little makes a mickle.

I do not participate in any social network so cannot share. However I am happy to share the URL for the page I used Siema in some heavily abusing way:) Here it is, though all in Russian: [Сulture Islands](https://www.cultureislands.ru/context/mamontov/).

In fact there are 2 instances of Siema on the page talking to each other and both controlling one another. You will see :) 

Thanks. :100:
